src/ks-app/tsconfig.json

```diff
@@ -16,8 +16,8 @@
       "dom"
     ],
     "paths": {
-//      "clarity-angular": [
-//        "../../dist/npm/clarity-angular"
+//      "clr-angular": [
+//        "../../dist/npm/clr-angular"
```

Comment:
What is this `//`?

Reply:
Not sure why that's was there. I removed it.